test/stdlib/tgmath.swift.gyb

```diff
@@ -53,6 +53,18 @@ func expectEqualWithTolerance<T>(_ expected: TestLiteralType, _ actual: T,
              file: file, line: line)
 }
 
+func checkGradient<T: BinaryFloatingPoint & Differentiable>(
+  _ f: @differentiable (T, T) -> T,
+  _ x: T,
+  _ y: T) where T == T.TangentVector {
```

Comment:
style nit: `) where T == T.TangentVector {` or `{` should be moved to the next line, with indentation level 0, to give visual separation between the argument list and the function body

Reply:
done in 7ce2e4c